### PR TITLE
refactor: consolidate logic formatting loan use

### PR DESCRIPTION
**Pull request by emuvente**
I need the full loan use statement for recommended loans in the portfolio overview page I'm working on, and I found that there wasn't a clear "correct" way to do that. I decided to take some time to cleanup the various differing methods of generating the same string and put everything behind a util function and local resolver `fullLoanUse`. I didn't touch the loan use in `KivaClassicBasicLoanCardExp`, given that it's just experiment code.

### CODE REVIEW

File: .storybook/mock-data/loan-data-mock.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ export default [
 				"__typename": "Image"
 			},
 			"anonymizationLevel": "none",
+			"fullLoanUse": "A loan of $600 helps to purchase heifers  to increase headcount of cattle and sales of organic milk.",
```
Comment (dyersituations) [inline .storybook/mock-data/loan-data-mock.js:63]: Looks like double space after "heifers"

File: src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue [hunk 2]
```diff
@@ -68,7 +68,7 @@
 						:name="loan.name"
 						:status="loan.status"
```
Comment (dyersituations) [inline src/components/LoanCards/HoverLoanCard/DetailedLoanCard.vue:69]: Looks like some of these props are no longer needed.

**Review by dyersituations — APPROVED** [review]
Looks good, some great cleanup as well! Checks failed with a linting error. I'll add a ticket to look into modifying the use string if the new loan card experiment works (or creating a variation in the util).

**Review by mcstover — APPROVED** [review]
Nice abstraction and bonus clean up!

File: .storybook/mock-data/loan-data-mock.js (hunk 1 above)
Comment (emuvente) [reply .storybook/mock-data/loan-data-mock.js:63]: Yeah, I saw that too, but that's in the original loan use statement so I preserved it.

File: src/components/BorrowerProfile/SummaryCard.vue [hunk 3]
```diff
@@ -57,7 +57,7 @@
 				</template>
 			</div>
 		</div>
-		<p class="tw-flex-none tw-w-full tw-mb-2 tw-text-h2">
+		<p class="tw-flex-none tw-w-full tw-mb-2 tw-text-h2" data-testid="bp-summary-loan-use">
```
Comment (mcstover) [inline src/components/BorrowerProfile/SummaryCard.vue:60]: Good catch!

**Pull request merged**